net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt

```diff
@@ -54,9 +62,9 @@ $( document ).ready(function () {
     ajaxCall(url="/api/lldpd/service/status", sendData={}, callback=function(data,status) {
         updateServiceStatusUI(data['status']);
     });
-    ajaxCall(url="/api/lldpd/service/neighbor", sendData={}, callback=function(data,status) {
```

Comment:
Zap the other one below too?

Reply:
How would it be called .. might be to stupid for this :)

Reply:
the update in the reconfigure block that I added and which started the whole discussion ;)

Reply:
oh ... ok, removed :)